packages/babel-plugin/__tests__/stylex-perf-create-theme-test.js

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
Can you explain what this test is for? I thought we already had unit tests for theme creation. This is a unit test, so doesn't benchmark anything. If we want a perf benchmark, we should add to the benchmarks instead.

Reply:
I added it as a test as I was trying to debug the root cause of the slowdown. It's not quite a benchmark, it's just a verification that the transformation isn't taking an unreasonably long amount of time.

I'm open to suggestions on how exactly to define this. It could be a benchmark, but I'm not sure what that would look like.

Reply:
No worries, I'll have a go at converting this into a benchmark next week